articles/azure-maps/elevation-data-services.md

```diff
@@ -0,0 +1,380 @@
+
+#### Results
+
+1. Select **Results >>** to view the tiles for the selected region and data set.
+
+1. The list of downloadable tiles appear on the results page. To download
+   only tiles you want, select the **Download Options** button on the result card for each tile,
+   selecting the option **GeoTIFF 1 Arc-Second** and repeat this step for the remaining tiles.
+
+    :::image type="content" source="./media/elevation-services/results-export.png" alt-text="A screenshot showing how to add raster tiles in QGIS." lightbox="./media/elevation-services/results-export.png":::
+
+1. Alternatively, use the bulk download option and select **GeoTIFF 1 Arc-second**.
+
+### Add raster tiles to QGIS
+
+Once you have the raster tiles you need, you can import them in QGIS.
+
+1. Add raster tiles to QGIS by dragging the files to the **QGIS layer**
+   tab or selecting **Add Layer** in the **Layer** menu.
+
+    :::image type="content" source="./media/elevation-services/add-raster-tiles-qgis.png" alt-text="A screenshot showing how to add raster tiles in QGIS." lightbox="./media/elevation-services/add-raster-tiles-qgis.png":::
+
+2. When the raster layers are loaded into QGIS, there can be
+   different shades of tiles. Fix this by merging the raster
+   layers, which result in a single smooth raster image in GeoTIFF
+   format. To do this, select **Miscellaneous** from the **Raster** menu, then **Merge...**
+
+    :::image type="content" source="./media/elevation-services/merge-raster-layers.png" alt-text="A screenshot showing how the merge raster layers menu in QGIS.":::
+
+3. Reproject the merged raster layer to EPSG:3857 (WGS84 / Pseudo-Mercator).
+   EPSG:3857 is required to use it with [Azure Maps Web SDK].
+
+    :::image type="content" source="./media/elevation-services/save-raster-layer.png" alt-text="A screenshot showing how the merge raster layers menu in QGIS.":::
+
+4. If you only want to create contour line vector tiles, you can skip the following steps and go to
+   [Create Contour line vector tile service using Azure Function and PostgreSQL].
+
+5. To create an Elevation API, the next step is to RGB-Encode the GeoTIFF. This can be done using
+    [rio-rgbify], developed by MapBox. There are some challenges running this tool directly in
+    Windows, so it's easier to run from WSL. Below are the steps in Ubuntu on WSL:
+
+    ```Ubuntu
+    sudo apt get update
+    sudo apt get upgrade
+    sudo apt install python3-pip
+    pip install rio-rgbify
+    PATH="$PATH:/home/<user  /.local/bin"
+    ```
+
+    The following steps are only necessary when mounting an external hard drive or USB flash drive:
+
+    ```Ubuntu
+    sudo mkdir /mnt/f
+    sudo mount -t drvfs D: /mnt/f
+    rio rgbify -b -10000 -i 0.1 wa_1arc_v3_merged_3857.tif wa_1arc_v3_merged_3857_rgb.tif
```

Comment:
Please ungroup the rio rgbify step from the mounting instructions since it is a required step.

Reply:
> Please ungroup the rio rgbify step from the mounting instructions since it is a required step.

    ```bash
    sudo apt get update
    sudo apt get upgrade
    sudo apt install python3-pip
    pip install rio-rgbify
    PATH="$PATH:/home/<user  /.local/bin"
    # The following two steps are only necessary when mounting an external hard drive or USB flash drive:
    sudo mkdir /mnt/f
    sudo mount -t drvfs D: /mnt/f

    rio rgbify -b -10000 -i 0.1 wa_1arc_v3_merged_3857.tif wa_1arc_v3_merged_3857_rgb.tif

    # The following steps are only necessary when unmounting an external hard drive or USB flash drive:
    cd \~
    sudo umount /mnt/f/
    ```